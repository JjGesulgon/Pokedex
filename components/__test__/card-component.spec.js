import { shallowMount } from "@vue/test-utils";
import CardComponent from "../card-component";


const factory = () => {
  return shallowMount(CardComponent, {
  });
};

describe("CardComponent", () => {
  test("mounts properly", () => {
    const wrapper = factory();
    expect(wrapper.exists()).toBeTruthy()
  });

  test("renders properly", () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});