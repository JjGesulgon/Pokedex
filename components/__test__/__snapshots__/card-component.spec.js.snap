// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardComponent renders properly 1`] = `<div class="grid grid-cols-2 md:grid-cols-5 gap-4 mt-16"></div>`;
